src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -856,13 +860,17 @@ bool FileConnection::checkDup(int fd)
 
 int FileConnection::openFile()
 {
-  JASSERT(jalib::Filesystem::FileExists(_path)) (_path)
+  dmtcp::string phys_path_string = "";
+  const char *phys_path =  virtual_to_physical_path ?
+                           virtual_to_physical_path(_path.c_str(), phys_path_string):
+                           _path.c_str();
+  JASSERT(jalib::Filesystem::FileExists(phys_path_string)) (_path)
```

Comment:
@gc00 @rohgarg : The bug is here I think. It's because the pathvirt plugin isn't loaded, so `phys_path_string` never gets reassigned from `""` and on line 685, it then asserts that the `""` exists as a file.